firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java

```diff
@@ -27,24 +27,27 @@
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.OutputStream;
 import java.io.UnsupportedEncodingException;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.zip.GZIPOutputStream;
```

Comment:
I see you now never compress and data. I think that's probably OK, since all our API calls (except the screenshot upload which does it's own compression) tend to have pretty small payloads.